README.md

```diff
@@ -30,7 +30,7 @@ materials science, quantum chemistry, and biology.
 
 ## Requirements
 
-DeepChem currently supports Python 3.5 through 3.7 and requires these packages on any condition.
+DeepChem currently supports Python 3.6 through 3.7 and requires these packages on any condition.

```

Comment:
DeepChem officially tested only Python 3.6 and 3.7, so I fixed.

Reply:
This isn't related to this PR, but I think with the latest RDKit DeepChem also works for 3.8. We need to add that into the test suite explicitly though